JSONResourceFile.js

```diff
@@ -161,23 +161,28 @@ JSONResourceFile.prototype._isPluralData = function(data) {
  * @private
  */
 JSONResourceFile.prototype._parsePluralData = function(data) {
-    var splitData = data.split("|");
     var parsePlural = {};
     var categoryMap = {
         "0" : "zero",
         "1" : "one",
         "2" : "two"
-    } 
+    }
+    var isPseudoString = false;
+    if (data.startsWith("[")) {
```

Comment:
This seems very fragile to me. If we change the definition of a pseudo string, then this will break. Is there some way that we can pass in the locale as the 2nd parameter?

`_parsePluralData = function(data, locale) {`

That way, we can check if it is a pseudo locale properly. (Or if the caller already knows, just add a parameter "isPseudoString" and make them pass it in.)

Reply:
Yes, Following your comments, I updated it.